Assets/HoloToolkit/Utilities/Scripts/TextToSpeechManager.cs

```diff
@@ -29,9 +56,13 @@ public class TextToSpeechManager : MonoBehaviour
         [Tooltip("The audio source where speech will be played.")]
         public AudioSource audioSource;
 
+        [Tooltip("The voice that will be used to generate speech.")]
+        public TextToSpeechVoice voice;
```

Comment:
NIT: This class does not follow the naming convention typically followed for public fields.

public TextToSpeechVoice Voice;